docs/rules/no-assignment-of-untracked-properties-used-in-tracking-contexts.md

```diff
@@ -69,6 +69,61 @@ class MyComponent extends Component {
 
 The autofixer for this rule will update assignments to use `set`. Alternatively, you can begin using tracked properties.
 
+## Configuration
+
+* object -- containing the following properties:
+  * array -- `extraMacros` -- array of configurations for custom macros, each with the following properties:
```

Comment:
What happens if someone provides a standard Ember computed property macro in this configuration? Can we disallow the Ember paths from being provided?

Reply:
I'll add a check for that.